src/main/java/org/folio/oaipmh/helpers/enrichment/ItemsHoldingsRequestWithDelayExecutor.java

```diff
@@ -0,0 +1,55 @@
+package org.folio.oaipmh.helpers.enrichment;
+
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.codec.BodyCodec;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.oaipmh.processors.OaiPmhJsonParser;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.folio.oaipmh.Constants.SKIP_SUPPRESSED_FROM_DISCOVERY_RECORDS;
+import static org.folio.oaipmh.helpers.AbstractGetRecordsHelper.INSTANCE_IDS_ENRICH_PARAM_NAME;
+
+public class ItemsHoldingsRequestWithDelayExecutor {
+
+  private static final Logger logger = LogManager.getLogger(ItemsHoldingsRequestWithDelayExecutor.class);
+
+  private final Vertx vertx = Vertx.currentContext().owner();
+  private final ItemsHoldingsEnrichment itemsHoldingsEnrichment;
+  private final AtomicInteger inc = new AtomicInteger(0);
```

Comment:
Default value in AtomicInteger is 0 anyway and, probably, it's not necessary to indicate it explicitly in constructor.

Reply:
Thanks, fixed.